database/src/main/java/database/UserRepository.java

```diff
@@ -0,0 +1,62 @@
+package database;
+
+import database.exception.FailedDataBaseCreationException;
+import game.player.User;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class UserRepository implements Repository<User> {
+
+    private static final String URL = "jdbc:h2:mem:db1";
+
+    private static final String TABLE_NAME = "users";
+
+    private static final String CREATE_TABLE_COMMAND =
+            "create table " + TABLE_NAME + "(name VARCHAR(255) PRIMARY KEY, id INT, objectId INT);";
+
+    private Statement statement;
+
+    public UserRepository() throws FailedDataBaseCreationException {
+        try {
+            Connection connection = DriverManager.getConnection(URL);
+
+            this.statement = connection.createStatement();
+            statement.execute(CREATE_TABLE_COMMAND);
+
+        } catch (SQLException e) {
+            throw new FailedDataBaseCreationException();
```

Comment:
"DataBase" word violates Repository at the same level as "SQL" word in SQLException. Nothing changed. 

```
throw new RepositoryException("Failed to create repository");
```